src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -209,7 +266,7 @@ CROSS APPLY OPENJSON([<refCol>]) WITH (Element VARCHAR(8000) '$') AS ARRAY_VALUE
       SELECT <columns:{c|tbl.[<c>]}; separator=",">
         FROM (SELECT row_number() over (order by <sort> <direction>) AS datarepo_row_number,
                      <columns:{c|rows.[<c>]}; separator=",">
-                FROM OPENROWSET(BULK 'parquet/*/<tableName>.parquet/*',
+                FROM OPENROWSET(BULK 'parquet/*/<tableName>/*.parquet/*',
```

Comment:
@nmalfroy  - I believe this is the query used for the preview endpoint. So, this would no longer work for old snapshots. I'll check if something like `'parquet/*/<tableName>*` works, but we might have to consider other solutions.